csrankings.csv

```diff
@@ -16153,3 +16153,37 @@ François Fleuret,EPFL,https://people.epfl.ch/francois.fleuret,Bj1tRlsAAAAJ
 Alexandre Alahi,EPFL,https://people.epfl.ch/alexandre.alahi,UIhXQ64AAAAJ
 Pierre Vandergheynst,EPFL,https://people.epfl.ch/pierre.vandergheynst,1p9NOFEAAAAJ
 Ali H. Sayed,EPFL,https://people.epfl.ch/ali.sayed,uDG9sXQAAAAJ
+Christian S. Jensen,Aalborg University,http://people.cs.aau.dk/~csj/,wo20cPYAAAAJ
+Kim G. Larsen,Aalborg University,www.cs.aau.dk/~kgl,neDFD60AAAAJ
+Torben Bach Pedersen,Aalborg University,http://people.cs.aau.dk/~tbp/,XGzPuKEAAAAJ
+Jesper Kjeldskov,Aalborg University,http://people.cs.aau.dk/~jesper/,pD3nfeAAAAAJ
+Hua Lu 0001,Aalborg University,http://people.cs.aau.dk/~luhua/,h1mExmcAAAAJ
+Peter Dolog,Aalborg University,https://peterdolog.wordpress.com/,gCafUI0AAAAJ
+Bin Yang 0002,Aalborg University,http://people.cs.aau.dk/~byang/,qjBQhoUAAAAJ
+Jirí Srba,Aalborg University,http://people.cs.aau.dk/~srba/,PYmuJQoAAAAJ
+Jan Stage,Aalborg University,http://people.cs.aau.dk/~jans/,6fgR7XcAAAAJ
+Mikael B. Skov,Aalborg University,http://people.cs.aau.dk/~dubois/,sivdLnwAAAAJ
+Kristian G. Olesen,Aalborg University,http://people.cs.aau.dk/~kgo/cv.html,qrPdfAcAAAAJ
+Simonas Saltenis,Aalborg University,http://people.cs.aau.dk/~simas/,K0l79_wAAAAJ
+Peter Axel Nielsen,Aalborg University,http://people.cs.aau.dk/~pan/,l2VkFywAAAAJ
+Bent Thomsen,Aalborg University,http://people.cs.aau.dk/~bt/,mhyMmZoAAAAJ
+Brian Nielsen,Aalborg University,http://people.cs.aau.dk/~bnielsen/,iAgWGgwAAAAJ
+Arne Skou,Aalborg University,http://people.cs.aau.dk/~ask/,qIjjvXwAAAAJ
+Thomas D. Nielsen,Aalborg University,http://people.cs.aau.dk/~tdn/,6fWF0CgAAAAJ
+Katja Hose,Aalborg University,http://people.cs.aau.dk/~khose/,LbXvZUoAAAAJ
+Jeni Paay,Aalborg University,http://people.cs.aau.dk/~jeni/jeni_homepage/home.html,-4miticAAAAJ
+Manfred Jaeger,Aalborg University,http://people.cs.aau.dk/~jaeger/,1E085gUAAAAJ,
+Hans Hüttel,Aalborg University,http://people.cs.aau.dk/~hans/index-eng.html,Wf7-agQAAAAJ
+René Rydhof Hansen,Aalborg University,http://people.cs.aau.dk/~rrh/,MGLEvwkAAAAJ
+Ulrik Nyman,Aalborg University,http://ulrik.blog.aau.dk/,Z3FXuI0AAAAJ
+Christian Thomsen,Aalborg University,http://personprofil.aau.dk/111293?lang=en,Y7i1yN4AAAAJ
+Lone Leth Thomsen,Aalborg University,http://people.cs.aau.dk/~lone/,vyYcihcAAAAJ
+Ivan Aaen,Aalborg University,http://people.cs.aau.dk/~ivan/,-8-9oRgAAAAJ
+Kristian Torp,Aalborg University,http://people.cs.aau.dk/~torp/,EusRZN0AAAAJ
+Kurt Nørmark,Aalborg University,http://people.cs.aau.dk/~normark/,5L7St-cAAAAJ
+Giorgio Bacci,Aalborg University,http://people.cs.aau.dk/~grbacci/,y2Q3kK8AAAAJ
+Anders Bruun,Aalborg University,http://personprofil.aau.dk/120220,X32A-C4AAAAJ

```

Comment:
I see a couple of people are not in CS but in "Institut for Datalogi". This doesn't sound like a CS department. Please explain how they meet the inclusion criteria. Thanks. A pointer to a web page from the CS department listing their faculty and affiliated faculty would be helpful.

Reply:
"Institut for Datalogi" is Danish and it is translated to CS department in English. Here is a link to CS 
https://www.cs.aau.dk/research/. In this link, you can find the staff members by click into different groups on this paper, e.g., DATABASE, PROGRAMMING AND WEB TECHNOLOGIES (DPW). Hope it is enough.

Reply:
In Denmark, "Department of Computer Science" is called "Institut for Datalogi". This also applies to the Danish departments already in csrankings. Here is a link to the department's website: https://www.cs.aau.dk/about/. This page describes the organization: https://www.cs.aau.dk/about/organisation/.  This one contains links to the three research groups, each of which lists the members of the group: https://www.cs.aau.dk/research/ (see a link “Staff” at the bottom of each of the three pages).

In Denmark, one needs to be a professor or an associate professor (lektor) to be the primary supervisor of a Ph.D. student. Thus, we only report staff in those two categories.